src/Index/IndexInterface.php

```diff
@@ -31,6 +31,14 @@ public function getName(): string;
      */
     public function getBatchSize(): int;
 
+    /**
+     * Defines if the the index should be populated in subprocesses.
+     * This is useful for large indexes to avoid memory issues.
+     *
+     * @return bool
+     */
+    public function shouldIndexInSubprocesses(): bool;
```

Comment:
The bundle calls this process "populating the index". Can you please rename the method to e.g. `shouldPopulateInSubprocesses`?

Reply:
👍